eldritch/items/__init__.py

```diff
@@ -0,0 +1,70 @@
+from eldritch.assets import Deputy
+from .base import Item, Weapon, OneshotWeapon, Tome
+from .common import (
+    AncientTome, Automatic45, Axe, DarkCloak,
+    Derringer18, Revolver38, Dynamite,
+    TommyGun, Food, ResearchMaterials,
+    Rifle, Shotgun, CavalrySaber, Whiskey, Knife, Lantern,
+    Bullwhip, Cross)
+
+from .unique import (
```

Comment:
My personal request for keeping this modular is to add a `__all__` to unique (and common, spells, and base) so that you can just say `from .unique import *` here. Though I suppose not deputy, as that's a special case.

Reply:
I've been told to shun `from x import *` in general, but I think here it really is the best way.

Reply:
Yeah, it's even discouraged by the python official documentation. But I do think it's warranted here.